### ISSUE TITLE
Sloth quit unexpectedly

### ISSUE THREAD

**Issue by McTTRS** [sveinbjornt/Sloth#25]
Translated Report (Full Report Below)
-------------------------------------

Process:               Sloth [5663]
Path:                  /private/var/folders/*/Sloth.app/Contents/MacOS/Sloth
Identifier:            org.sveinbjorn.Sloth
Version:               3.2 (300)
Code Type:             ARM-64 (Native)
Parent Process:        launchd [1]
User ID:               501

Date/Time:             2023-02-20 15:54:36.2630 -0600
OS Version:            macOS 13.2.1 (22D68)
Report Version:        12
Anonymous UUID:        86E7B0F0-BD51-11DB-D066-BFD4E810BCEE

Time Awake Since Boot: 37000 seconds

System Integrity Protection: enabled

Notes:
dyld_process_snapshot_create_for_process failed with 5

Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_BAD_ACCESS (SIGBUS)
Exception Codes:       KERN_PROTECTION_FAILURE at 0x000000016f254378
Exception Codes:       0x0000000000000002, 0x000000016f254378

Termination Reason:    Namespace SIGNAL, Code 10 Bus error: 10
Terminating Process:   exc handler [5663]

VM Region Info: 0x16f254378 is in 0x16f254000-0x16f258000;  bytes after start: 888  bytes before end: 15495
      REGION TYPE                    START - END         [ VSIZE] PRT/MAX SHRMOD  REGION DETAIL
      Stack                       16ea58000-16f254000    [ 8176K] rw-/rwx SM=PRV  thread 0
--->  STACK GUARD                 16f254000-16f258000    [   16K] ---/rwx SM=NUL  ... for thread 1
      Stack                       16f258000-16f2e0000    [  544K] rw-/rwx SM=PRV  thread 1

Error Formulating Crash Report:
dyld_process_snapshot_create_for_process failed with 5

**Comment by sveinbjornt** [comment]
I am neither on arm64 nor Ventura yet, so am unable to replicate. But thank you for your feedback!

**Comment by weiran** [comment]
I also get a crash on startup with Ventura. Running Intel not ARM:

```
-------------------------------------
Translated Report (Full Report Below)
-------------------------------------

Process:               Sloth [19801]
Path:                  /Applications/Sloth.app/Contents/MacOS/Sloth
Identifier:            org.sveinbjorn.Sloth
Version:               3.2 (300)
Code Type:             X86-64 (Native)
Parent Process:        launchd [1]
User ID:               501

Date/Time:             2023-02-21 10:26:58.1490 +0000
OS Version:            macOS 13.2 (22D49)
Report Version:        12
Anonymous UUID:        9FD0212C-D889-EA88-A188-93481055B559

Sleep/Wake UUID:       7D30FDA3-D32C-4DC0-8422-33E90456639E

Time Awake Since Boot: 2200000 seconds
Time Since Wake:       1086705 seconds

System Integrity Protection: enabled

Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_BAD_ACCESS (SIGSEGV)
Exception Codes:       KERN_INVALID_ADDRESS at 0x00007ff7b86a2260
Exception Codes:       0x0000000000000001, 0x00007ff7b86a2260

Termination Reason:    Namespace SIGNAL, Code 11 Segmentation fault: 11
Terminating Process:   exc handler [19801]

VM Region Info: 0x7ff7b86a2260 is not in any region.  Bytes after previous region: 609  Bytes before following region: 1301101984
      REGION TYPE                    START - END         [ VSIZE] PRT/MAX SHRMOD  REGION DETAIL
      Stack                    7ff7b7ea2000-7ff7b86a2000 [ 8192K] rw-/rwx SM=PRV  thread 0
--->  GAP OF 0x4d8d4000 BYTES
      unused __TEXT            7ff805f76000-7ff805fce000 [  352K] r-x/r-x SM=COW  ...ed lib __TEXT

Thread 0 Crashed::  Dispatch queue: com.apple.main-thread
0   libsystem_platform.dylib      	    0x7ff80636b572 _platform_strlen + 18
1   Foundation                    	    0x7ff80723e790 +[NSString stringWithUTF8String:] + 29
2   Sloth                         	       0x107870c25 0x10785e000 + 76837
3   ???                           	      0x1a00000000 ???
```

**Comment by YBelikov** [comment]
Hi!
I also experienced similar problem
Here is a part of the crash report:
`-------------------------------------
Translated Report (Full Report Below)
-------------------------------------

Process:               Sloth [15339]
Path:                  /Applications/Sloth.app/Contents/MacOS/Sloth
Identifier:            org.sveinbjorn.Sloth
Version:               3.3 (308)
Code Type:             ARM-64 (Native)
Parent Process:        launchd [1]
User ID:               501

Date/Time:             2023-02-22 18:35:06.4811 +0200
OS Version:            macOS 13.1 (22C65)
Report Version:        12
Anonymous UUID:        4860F4A7-6C81-9D72-DA08-077A3C3E20F0

Sleep/Wake UUID:       103D6A14-7135-4517-8799-89490648AA1F

Time Awake Since Boot: 20000 seconds
Time Since Wake:       362 seconds

System Integrity Protection: disabled

Crashed Thread:        1  Dispatch queue: com.apple.root.user-initiated-qos

Exception Type:        EXC_BAD_ACCESS (SIGBUS)
Exception Codes:       KERN_PROTECTION_FAILURE at 0x000000016ee14270
Exception Codes:       0x0000000000000002, 0x000000016ee14270`

However this problem appeared when I tried to launch app that I have built locally. The version downloaded through Homebrew works fine.
